packages/wallets/wallet-turnkey/src/strategy/strategy.ts

```
@@ -0,0 +1,739 @@
+        this.turnkey.getSession(),
+        this.authIframeClient.getWhoami(),
+      ])
+
+      const organizationId =
+        user?.organizationId || session?.organizationId || this.organizationId
+
+      if (loginResult) {
+        this.setStatus(TurnkeyStatus.LoggedIn)
+        this.organizationId = organizationId
+        // TODO: for some reason this is writing the incorrect organizationId
+        if (session && session?.expiry) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)